guide/src/tutorials/local-chains/relay-paths/multiple-paths.md

```diff
@@ -4,95 +4,82 @@ Hermes can relay packets over all current or future paths between the configured
 
 Follow the steps below to connect three chains together and relay packets between them:
 
-1. Paste the following configuration in the standard Hermes configuration file at `~/.hermes/config.toml`:
+add to gm.toml
+gm start
+gm hermes config
+gm hermes keys
+gm status
```

Comment:
These steps get rendered all in a single line, which I don't think is the intention. Are these steps meant to replace the numbered steps below?

Reply:
This seems to be a typo. Will fix it.